bundle/edu.gemini.p2checker/src/test/scala/edu/gemini/p2checker/rules/GmosSDtaXRuleTest.scala

```diff
@@ -71,6 +78,11 @@ class GmosSDtaXRuleTest extends AbstractRuleTest {
 
   private def expectFailure2(): Unit = expectFailure("Ybin=2")
```

Comment:
Maybe instead of 1, 2, 3 we could pass the `Set` of problem ids we aren't expecting to `expectSuccess` and set of problem ids we are expecting to `expectProblem`.